app/Http/Controllers/CustomTourController.php

```diff
@@ -54,6 +50,36 @@ public function show(Request $request, $id)
         ]);
     }
 
+    public function pdfLayout(Request $request, $id)
+    {
+        $customTourItem = CustomTour::find($id);
```

Comment:
The variable names this a custom tour item, but the model it references is a `CustomTour` and not `CustomTourItem`, so it's a bit confusing as to what this is supposed to be retrieving.

Reply:
This has been addressed.